src/Microsoft.Azure.WebJobs.Host/Indexers/FunctionIndexer.cs

```diff
@@ -109,7 +112,13 @@ internal class FunctionIndexer
                 }
                 catch (FunctionIndexingException fex)
                 {
+                    if (_allowPartialHostStartup)
+                    {
+                        fex.Handled = true;
```

Comment:
When this mode is set, we'll now be defaulting these to handled. In Functions currently we're setting this manually (code [here](https://github.com/Azure/azure-functions-host/blob/749564ab175f7c0d2e4ef811cb33f354e460803f/src/WebJobs.Script/Host/ScriptHost.cs#L1800)). We'll be able to remove that in favor of setting the new host config property.